Homework-№1-from-07.09.2022/task2.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdbool.h>
+
+int division(int firstNum, int secondNum)
+{
+    int incomQuot = 0;
+    while (firstNum >= secondNum)
+    {
+        firstNum -= secondNum;
+        incomQuot += 1;
+    }
+    return incomQuot;
+}
+
+bool remainderExist(int firstNum, int secondNum, int incomQuot)
+{
+    int remainder = firstNum - secondNum * incomQuot;
+    return (remainder != 0);
+}
+
+int main(){
+    int firstNum, secondNum, answer = 0, signOfResult = 0, dop1 = 0;

```

Comment:
```suggestion
    int firstNum, secondNum, answer = 0, signOfResult = 0;
```
Непонятная имя переменной. Либо удали её (а без нее вполне можно обойтись, если сразу прибавлять вычисленное значение к ответу.
Либо название более содержательное придумай. Что-то вроде 'additionalValueByRemainder'